src/app/Scenes/Artwork/Artwork.tsx

```diff
@@ -63,6 +65,7 @@ export const Artwork: React.FC<ArtworkProps> = ({
 }) => {
   const [refreshing, setRefreshing] = useState(false)
   const [fetchingData, setFetchingData] = useState(false)
+  const conversationalBuyNow = useFeatureFlag("AREnableConversationalBuyNow")
```

Comment:
(non-blocking): I think we use the following convention everywhere:

```suggestion
  const enableConversationalBuyNow = useFeatureFlag("AREnableConversationalBuyNow")
```

Reply:
good to know, danke! changed in https://github.com/artsy/eigen/pull/6678/commits/15a5e3279a51a2dd5c18a55dc9fd36288219e99e